NuRadioMC/simulation/simulation.py

```diff
@@ -1861,3 +1863,21 @@ def integrated_channel_response(self):
     @integrated_channel_response.setter
     def integrated_channel_response(self, value):
         self._integrated_channel_response = value
+
+def _calculate_amp_per_ray_solution(station):
+    """ Calculate the max amplitude and time of the ray solutions
+
+    We are using this function instead of the channelSignalReconstructor
```

Comment:
Can you be a bit more verbose in your description here. We don't want people to use this function instead of channelSIgnalReconstructor all the time.